frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/create-local-volume-set.tsx

```diff
@@ -0,0 +1,256 @@
+              value="allNodes"
+              onChange={toggleShowNodesList}
+              description="Selecting all nodes will search for available volume storage on all nodes."
+              defaultChecked
+            />
+            <Radio
+              label="Select nodes"
+              name="nodes-selection"
+              id="create-lvs--radio-select-nodes"
+              value="selectedNodes"
+              onChange={toggleShowNodesList}
+              description="Selecting nodes allow you to limit the search for available volumes to specific nodes."
+            />
+          </div>
+        </FormGroup>
+        {showNodesList && (
+          <ListPage
+            customData={{ rows, setRows, allSelected, setAllSelected }}
+            showTitle={false}
+            kind="Node"
+            ListComponent={NodesSelectionList}
+          />
+        )}
+        <FormGroup label="Volume Type" fieldId="create-lvs--volume-type-dropdown">
+          <Dropdown
+            id="create-lvs--volume-type-dropdown"
+            dropDownClassName="dropdown--full-width"
+            items={volumeTypeDropdownItems}
+            title={VOLUME_TYPE[volumeType].title}
+            selectedKey={volumeType}
+            onChange={setVolumeType}
+          />
+        </FormGroup>
+        <Expandable toggleText="Advanced">
+          <FormGroup
+            label="Volume Mode"
+            fieldId="create-lso--volume-mode-dropdown"
+            className="lso-create-lvs__volume-mode-dropdown--margin"
+          >
+            <Dropdown
+              id="create-lso--volume-mode-dropdown"
+              dropDownClassName="dropdown--full-width"
+              items={volumeModeDropdownItems}
+              title={VOLUME_MODE[volumeMode].title}
+              selectedKey={volumeMode}
+              onChange={setVolumeMode}
+            />
+          </FormGroup>
+          <FormGroup label="Max Volume Limit" fieldId="create-lvs--max-volume-limit">
+            <p className="help-block lso-create-lvs__max-volume-limit-help-text--margin">
+              Volume limit will set the maximum number of PVs to create on a node. If the field is
+              empty, will create PVs for all available volumes on the matching nodes.
+            </p>
+            <TextInput
+              type={TextInputTypes.number}
+              id="create-lvs--max-volume-limit"
+              value={maxVolumeLimit}
+              onChange={setMaxVolumeLimit}
+            />
+          </FormGroup>
+        </Expandable>
+        <ButtonBar errorMessage={errorMessage} inProgress={inProgress}>
+          <ActionGroup>
+            <Button data-test-id="create-lvs-form-create-button" type="submit" variant="primary">
+              Create
+            </Button>
+            <Button
+              data-test-id="create-lvs-form-cancel-button"
+              type="button"
+              variant="secondary"
+              onClick={onCancel}
+            >
```

Comment:
Button's really don't need data-test-id since we have a common class for `Primary` used throughout the console.

Reply:
I added as per your suggestion on chat, will remove again.